### PR TITLE
feat(table): add filters

### LINKED ISSUE #55: Repaid By
Repaid by is not relevant pool , its for a specific loan 

Image

### LINKED ISSUE #128: Add font family Tooltip component

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- Resolve #55 
- Resolve #128

### Steps to test

1. Run project
2. Check changes
 

Now:
imagen
imagen
imagen
imagen

#### CheckList

- [X] Follow proper Markdown format
- [X] Content is available in all languages
- [X] The content is adequate
- [X] I Ran a spell check
- [X] I included test coverage

### CODE REVIEW

File: src/stories/Table.stories.tsx [hunk 1]
```diff
@@ -23,6 +29,7 @@ const Template: Story<ITable> = (args) => {
       rowsPage={[5, 10, 15]}
       rowsNumberLabel="Rows per page:"
       handleSelectedRowsPage={() => {}}
+      setFilters={SetData}
```
Comment (leisterfrancisco) [inline src/stories/Table.stories.tsx:32]: should `SetData` use Pascal Case?

File: src/components/Table/index.tsx [hunk 2]
```diff
@@ -69,6 +81,16 @@ export const Table = ({
     return paginationArray;
   };
 
+  const updateDate = () => {
```
Comment (leisterfrancisco) [inline src/components/Table/index.tsx:84]: Why Date? isn't it Data?

**Review by leisterfrancisco — APPROVED** [review]

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
please, see my comment, by mistake I clicked approve instead of request

File: src/stories/Table.stories.tsx [hunk 3]
```diff
@@ -11,6 +11,12 @@ export default {
 };
 
 const Template: Story<ITable> = (args) => {
+  const [data, SetData] = useState<String[]>([]);
+
+  useEffect(() => {
+    console.log(data);
+  }, [data]);
```
[the remark this answers is not in the document]
Comment (JAMares) [reply src/stories/Table.stories.tsx:18]: This PR is current WIP, is for test

File: src/components/Table/index.tsx (hunk 2 above)
Comment (JAMares) [reply src/components/Table/index.tsx:84]: Fixed it

**Review by Edwinjrc — APPROVED** [review]
LGTM

File: src/stories/Table.stories.tsx [hunk 4]
```diff
@@ -1,16 +1,86 @@
...
-import { Table, ITable } from "../components/Table";
-import optionIcon from "../../public/assets/options-icon.svg";
+import { Table, ITable, IRowsObject } from "../components/Table";
 
 export default {
   title: "Table",
   component: Table,
 };
 
+const dataTable = [
```
Comment (leisterfrancisco) [inline src/stories/Table.stories.tsx:12]: rename it to tableData

**Review by leisterfrancisco — APPROVED** [review]

**Comment by xavier506** [comment]
@JAMares  lets add a date selector a a multiple selection droprdown for status filters

File: src/components/Table/index.tsx [hunk 5]
```diff
@@ -18,6 +18,7 @@ export interface IHeaderObject {
 export interface IFilterObject {
   label: string;
   options?: Array<String>;
+  type?: "multiple" | "date";
```
Comment (leisterfrancisco) [inline src/components/Table/index.tsx:21]: since it can only be multiple or date, should it be optional?

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: src/stories/DropDown.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+export default {
+  title: "DropDown",
+  component: DropDown,
+};
+
+const Template: Story<IDropDownObject> = (args) => <DropDown {...args} />;
+
+const names = [
+  "Oliver Hansen",
```
Comment (xavier506) [inline src/stories/DropDown.stories.tsx:14]: What is this list of names?

**Comment by leisterfrancisco** [comment]
@JAMares this interface

export interface IFilterObject {
  label: string;
  options?: string[];
  type?: "multiple" | "date"; 
}

`type` should not be optional

**Pull request merged**